### PR TITLE
Bumping goreleaser to latest version in 0.31.x line

### PR DESCRIPTION
**Pull request by sethiyash**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -19,14 +19,23 @@ jobs:
...
+        id: version
+
       - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@5e15885530fb01d81d1f24e8a6f54ebbd0fed7eb
+        # GoReleaser v4.2.0
+        uses: goreleaser/goreleaser-action@f82d6c1c344bcacabba2c841718984797f664a6b
         if: startsWith(github.ref, 'refs/tags/')
         with:
-          version: 0.181.1
+          version: 0.16.2
```
Comment (joaopapereira) [inline .github/workflows/release.yml]:
```suggestion
          version: 1.16.2
```

**Review by joaopapereira — CHANGES_REQUESTED** [review]
I think the goreleaser version is incorrect

File: .github/workflows/release.yml (hunk 1 above)
Comment (sethiyash) [reply .github/workflows/release.yml]: Good Catch! Thanks, Joao.

**Review by joaopapereira — APPROVED** [review]
LGTM

**Pull request merged**